### PR TITLE
[TLP] how-to-contribute remove incubator

### PR DESCRIPTION
**Pull request by hf200012**
# Proposed changes

[TLP] how-to-contribute remove incubator

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/community/how-to-contribute/pull-request.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ under the License.
 
 ### 1. Fork仓库
 
-进入 apache/incubator-doris 的 [github 页面](https://github.com/apache/incubator-doris) ，点击右上角按钮 `Fork` 进行 Fork。
+进入 apache/doris 的 [github 页面](https://github.com/apache/incubator-doris) ，点击右上角按钮 `Fork` 进行 Fork。
```
Comment (morningman) [inline docs/zh-CN/community/how-to-contribute/pull-request.md:33]:
```suggestion
进入 apache/doris 的 [github 页面](https://github.com/apache/doris) ，点击右上角按钮 `Fork` 进行 Fork。
```

File: docs/zh-CN/community/how-to-contribute/pull-request.md [hunk 2]
```diff
@@ -250,9 +250,9 @@ Total 6 (delta 4), reused 0 (delta 0)
 remote: Resolving deltas: 100% (4/4), completed with 4 local objects.
 remote: 
 remote: Create a pull request for 'my_branch' on GitHub by visiting:
-remote:      https://github.com/lide-reed/incubator-doris/pull/new/my_branch
+remote:      https://github.com/lide-reed/doris/pull/new/my_branch
 remote: 
-To https://lide-reed:fc35ff925bd8fd6629be3f6412bacee99d4e5f97@github.com/lide-reed/incubator-doris.git
+To https://lide-reed:fc35ff925bd8fd6629be3f6412bacee99d4e5f97@github.com/lide-reed/doris.git
```
Comment (morningman) [inline docs/zh-CN/community/how-to-contribute/pull-request.md:255]: fc35ff925bd8fd6629be3f6412bacee99d4e5f97

This is a private token, delete it or desensitize it.

Comment (hf200012) [reply docs/zh-CN/community/how-to-contribute/pull-request.md]: OK

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**